javascript/react-verbs/src/verbs/Spread/Spread.tsx

```diff
@@ -20,80 +19,73 @@ import type { StepComponentProps } from '../../types.js'
 /**
  * Provides inputs for a step that needs lists of columns.
  */
-export const Spread: React.FC<StepComponentProps> = memo(function Spread({
-	step,
-	store,
-	table,
-	onChange,
-	input,
-}) {
-	const internal = useMemo(() => step as SpreadStep, [step])
+export const Spread: React.FC<StepComponentProps<SpreadArgs>> = memo(
```

Comment:
Have you gotten rid of the Step aliases entirely in favor of specifying the args type (e.g., `SpreadStep = Step<SpreadArgs>`)?

Reply:
Yeah, just to tighten things down. I'm not sure the XStep types were useful

Reply:
Yes, in an effort to streamline the exported surface area of Core. I didn't find much utility in the Step aliases.